### PR TITLE
Fix: validate kyc address upload min, max file size

### PR DESCRIPTION
**Pull request by seanyellowd**
https://openware-inc.atlassian.net/browse/OPSM-2249

The FE size limits are depend on the values of the next parameters in the barong configs:
    barong_upload_size_min_range
    barong_upload_size_max_range

### CODE REVIEW

File: web/mocks/api/v2/barong/resource/users/me/GET.mock [hunk 1]
```diff
@@ -76,5 +76,7 @@ access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Au
             "updated_at": "2019-05-16T14:01:49Z"
         }
     ],
-    "phones": "+380501823918"
+    "phones": "+380501823918",
+    "barong_upload_size_min_range": 0,
+    "barong_upload_size_max_range": 20
```
Comment (akhlopiachyi) [inline web/mocks/api/v2/barong/resource/users/me/GET.mock:81]: Why did you add these values to response of /users/me?

Comment (josadcha) [inline web/mocks/api/v2/barong/resource/users/me/GET.mock:81]: AFAIK API users/me doesn't return `barong_upload_size_min_range` and `barong_upload_size_max_range` values
the trick of this ticket is also make the barong part to return this information for user

Comment (seanyellowd) [reply web/mocks/api/v2/barong/resource/users/me/GET.mock:81]: I will move it to config.

Comment (seanyellowd) [reply web/mocks/api/v2/barong/resource/users/me/GET.mock:81]: Moved to config

Comment (seanyellowd) [reply web/mocks/api/v2/barong/resource/users/me/GET.mock:81]: Moved to config

**Review by josadcha — APPROVED** [review]

**Comment by josadcha** [comment]
@seanyellowd please fix sonar cloud issue: https://sonarcloud.io/project/issues?id=openware_baseapp&pullRequest=641&resolved=false&types=CODE_SMELL

**Comment by josadcha** [comment]
also, please ask Ihor QA to test the image (don't forget to ask him to set envs - min and max size on the deploynent)
image: `67f288b`

**Review by akushniruk — APPROVED** [review]
